Dependencies/UrlLib/Source/Windows/UrlRequest.cpp

```diff
@@ -164,6 +170,29 @@ namespace UrlLib
             return m_responseString;
         }
 
+        std::string ResponseHeader(const std::string& headerName) const
+        {
+            const auto iter = m_headers.find(headerName);
+            if (iter != m_headers.end())
+            {
+                return iter->second;
+            }
+            return {};
+        }
+
+        std::string ResponseHeaders() const
+        {
+            std::string str{};
+            for (auto& iter : m_headers)
+            {
+                str += iter.first;
+                str += ":";
+                str += iter.second;
+                str += "\n";
+            }
+            return str;
```

Comment:
Should use `std::stringstream` to be more efficient.

Also, I noted below the contract for this API must be able to return `null` and filter out `Set-Cookie`.

Reply:
I think maybe we should hold off on implementing this function since Babylon.js doesn't use it. If we want to add it, we should at least add tests.

Reply:
I'm removing the method.